### ISSUE TITLE
[BUG]Cellpose is not recognising NPY file

### ISSUE THREAD

**Issue by Yunxia-wang** [MouseLand/cellpose#1282]
**Describe the bug**
Hi. I annotated some images on one computer (#1), and I’m trying to train Cellpose SAM using these annotations on another computer (#2). (I annotated multiple images before training, because my computer’s GPU is not compatible with PyTorch, so I need to use my CPU, which takes between 1-3 hours to train a new model, versus just 2-5 minutes to run the model).

Cellpose on computer #2 is not recognising the .npy files from the annotations of computer #1, shows error message “ERROR: not NPY”.

On computer #1, I can open .npy files annotated on computer #2. But once I save the same files using Cellpose on computer #1, I can no longer open them on computer #2.

Both computers are using the same Cellpose version (4.0.4) and Numpy version (1.24.4), but computer #1 uses a higher version of Python and Torch.

Why does this happen? Is there a simple fix so that I can open the images on computer #2?

Cellpose version on computer #1

<img width="874" height="165" alt="Image" src="https://github.com/user-attachments/assets/475539ed-909a-434a-87ab-e567dd4a9ec2" />

Cellpose version on computer #2 

<img width="653" height="140" alt="Image" src="https://github.com/user-attachments/assets/63fd7db6-6e66-4953-b363-6f57432abb8a" />

Error message when trying to open .npy file.

<img width="132" height="18" alt="Image" src="https://github.com/user-attachments/assets/1d90860d-6cfc-442a-81e1-81d4a282368e" />

I just want to check it with a second computer. My main concern is whether the ._seq.npy generated by the first computer can be used to retrain the model in a linux system. I want to train a new model for cell segmentation using the manually annotated results.

**Comment by mrariden** [comment]
Can you upload the numpy files from both computers? I've never heard of this issue before. 

For your workflow at the moment, you can save the annotated masks as .tif or .png and avoid using .npy. It shouldn't matter for training, and it's easier to check that the data is the same using an image viewer.

**Comment by mrariden** [comment]
closing due to inactivity
